### PR TITLE
Added Games Section

### PR DESCRIPTION
**Pull request by abishekvashok**
Completed/Resolved?Solved issue https://github.com/fossasia/gci15.fossasia.org/issues/377 , to add Games section in FOSSASIA website for GCI 2015/2016 as a part of Google Code-in

### CODE REVIEW

**Comment by prwatch** [comment]
DEMO: http://Abhi2424shek.github.io/gci15.fossasia.org

**Comment by roonyh** [comment]
Thanks! Trying on http://Abhi2424shek.github.io/gci15.fossasia.org looks like I can't go through the items in the Gallery section anymore.

There should be a title identifying the sections as "Games"

File: index.html [hunk 1]
```diff
@@ -1,5 +1,3 @@
----
----
```
Comment (roonyh) [inline index.html:2]: Don't remove this. It might not look it but this is very important.

**Comment by roonyh** [comment]
@Abhi2424shek you can open the link http://Abhi2424shek.github.io/gci15.fossasia.org to see how your code would look.

**Pull request closed without merging**